proarc-common/src/main/java/cz/cas/lib/proarc/common/fedora/LocalStorage.java

```diff
@@ -179,6 +179,11 @@ public String asText() throws DigitalObjectException {
         }
 
         @Override
+        public void purgeDatastream(String datastream, String logMessage) throws DigitalObjectException {
+            RemoteStorage.getInstance().find(getPid()).purgeDatastream(datastream, logMessage);
```

Comment:
Tohle je abstrakce lokálního objektu uloženého na fs nebo jen v paměti. Volat mazání na RemoteStorage přece nejde. Ten PID tam ještě neexistuje. Patrně chybí test pokud to prošlo.

Reply:
Jakým způsobem by se tedy `purgeDatastream` měl řešit v případě `LocalStorage`, pokud jej není možné volat na `FedoraClient`?

Reply:
LocalStorage je abstrakce nad filesystemem a LocalObject je abstrakce nad FOXML, které je také součástí FedoraClient API (DigitalObject). Manipulace se provádí pomocí FoxmlUtils.
Pokud akce na mazání streamu není v části UI Import, tak může implementace zůstat jako UnsupportedOperationEx. Určitě ji nelze přesměrovat do RemoteRepository. Ve fázi Import odpovídající RemoteObject většinou neexistuje.

Reply:
Kontroloval jsem importní UI a akce pro smazání DS není k dispozici, tedy UnsupportedOpEx by mělo stačit.

Reply:
OK